tests/e2e/pageObjects/browser-page.ts

```diff
@@ -1139,7 +1142,7 @@ export type SortedSetKeyParameters = {
  * @param keyName The name of the key
  * @param element The element in list
  */
-export type ListKeyParameters = {
+export type ListAndStringKeyParameters = {
```

Comment:
I don't think that this is a good practice to combine two types in one
for list key we have  keyName and  element
for string key we have keyName and value